string layer_name
string class_name
string modifier
geometry_msgs/Pose2D pose
---
string class_name
float64 x
float64 y
